reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -528,12 +495,17 @@ def restart(self):
         """
         raise NotImplementedError()
 
-    def down(self):
+    def down(self, delete_traefik=False, delete_secrets=False):
         """Bring REANA cluster down, i.e. deletes all deployed components.
 
         Deletes all Kubernetes Deployments, Namespaces, Resourcequotas and
         Services that were created during initialization of REANA cluster.
 
+        :param delete_traefik: Wheter REANA traefik should be deleted or not.
```

Comment:
```suggestion
        :param delete_traefik: Whether REANA traefik should be deleted or not.
```

Also happening two lines below.